src/components/pages/Onboarding.tsx

```diff
@@ -8,18 +9,33 @@ type OnboardingProps = {
 export default class Onboarding extends React.Component<OnboardingProps> {
   render() {
     return (
-      <View style={styles.container}>
-        <Text>This is the onboardibng page.</Text>
+      <View style={[styles.slide, { backgroundColor: "#C04DEE" }]}>
+        <Text style={styles.header}>Give Raha</Text>
+        <Text style={styles.text}>
+          By using Raha, you are supporting basic income for everyone.
```

Comment:
What's the copy for swipe pages? How about
1. Raha is a payments network where transaction fees fund a basic income for all members.
1. To create a trusted network and prevent fraud and theft, Raha is invite only and all members must verify identity via video.
1. Create your video today to join Raha and join an economy where we can help each other and recognize every life has value.
Also I like what "Shop your Way" does where you can click/swipe next or just hit "Get Started" to skip over